src/pipeline.cpp

```diff
@@ -743,12 +743,13 @@ PIC.addClassToPassName(decltype(CREATE_PASS)::name(), NAME);
     }
 }
 
-NewPM::NewPM(std::unique_ptr<TargetMachine> TM, OptimizationLevel O, OptimizationOptions options) :
+NewPM::NewPM(std::unique_ptr<TargetMachine> TM, OptimizationLevel O, LLVMContext &ctx, OptimizationOptions options) :
     TM(std::move(TM)),
 #if JL_LLVM_VERSION < 160000
     SI(false),
     PIC(createPIC(SI)),
 #else
+    SI(ctx, false),
     PIC(createPIC()),
 #endif
```

Comment:
```suggestion
#if JL_LLVM_VERSION < 160000
    SI(false),
#else
    SI(ctx, false),
#endif
    PIC(createPIC(SI)),
```

I think now PIC can take SI again?

Reply:
Oh, forgot to fix the thing that made me do the whole PR :)